aea/cli_gui/__main__.py

```diff
@@ -23,21 +23,21 @@
 
 import aea.cli_gui
 
-parser = argparse.ArgumentParser(description="Launch the gui through python")
+parser = argparse.ArgumentParser(description="Launch the gui through python")  # pragma: no cover
 parser.add_argument(
     "-p", "--port", help="Port that the web server listens on", type=int, default=8080
-)
+)  # pragma: no cover
 
 parser.add_argument(
     "-H",
     "--host",
     help="host that the web server serves from",
     type=str,
     default="127.0.0.1",
-)
+)  # pragma: no cover
 
-args, unknown = parser.parse_known_args()
+args, unknown = parser.parse_known_args()  # pragma: no cover
 
 # If we're running in stand alone mode, run the application
-if __name__ == "__main__":
+if __name__ == "__main__":  # pragma: no cover
     aea.cli_gui.run(args.port, args.host)  # pragma: no cover
```

Comment:
Remove the `# pragma: no cover` from here.  You don't need it since you are using it on the if statement.